packages/utils/src/string.ts

```diff
@@ -38,7 +38,7 @@ const JS_PRIMITIVE_TYPES = [
   'string',
   'any',
   'Array',
-  /^Record<(.+),(.+)>$/,
+  // /^Record<(.+),(.+)>$/,
```

Comment:
We should change this to `Record`
```suggestion
  'Record',
```